benchmarks-next/meshoptimizer/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu:18.04 AS builder
+WORKDIR /
+RUN apt update && apt install -y wget git build-essential
+
+# Download and extract wasi-sdk.
+RUN wget https://github.com/WebAssembly/wasi-sdk/releases/download/wasi-sdk-12/wasi-sdk-12.0-linux.tar.gz
+RUN tar xvf wasi-sdk-12.0-linux.tar.gz
+
+WORKDIR /usr/src
+RUN git clone https://github.com/jlb6740/meshoptimizer.git \
+    && cd meshoptimizer \
+    && git checkout sightglass
+
+# Second, compile the benchmark to Wasm.
+
+WORKDIR /usr/src/meshoptimizer
+#COPY benchmark.c .
```

Comment:
This can be removed, I think.

Reply:
> It looks like these are the benchmarks that end up different after rebuilding:
> 
> * `benchmarks-next/meshoptimizer/benchmark.wasm`
> * `benchmarks-next/pulldown-cmark/benchmark.wasm`
> 
> The first is interesting. Did you commit the most up-to-date build? Otherwise this points at nondeterministic builds, which isn't great. We should probably [attach the built files as artifacts](https://github.com/bytecodealliance/wasmtime/blob/main/.github/workflows/main.yml#L205-L209) from the CI job so that we can download them and commit them without having to actually rebuild locally ourselves. Then if the CI job fails on re-push we know that there is non-determinism rather than just out-of-date `.wasm` binaries committed.
> 
> I'm not sure why the `pulldown-cmark` benchmark is apparently being nondeterministic, but I think @abrown observed this before. I _think_ there might be a hash map used in the `build.rs` script for that crate or something. I think we can ignore it for now and investigate further in follow ups.

Ok, thanks! When I resubmit I'll see if these go away. Perhaps for meshoptimizer I did submit an older version. Really I just submitted this to see if it would pass CI so expected a few clean-up items.